templates/dag-deploy/dag-server-statefulset.yaml

```diff
@@ -2,6 +2,7 @@
 ## dag-server StatefulSet ##
 ############################
 {{- if .Values.dagDeploy.enabled }}
+{{- $persistenceEnabled := .Values.dagDeploy.persistence.enabled }}
```

Comment:
Why do we need this? Why can't we just use `.Values.dagDeploy.persistence.enabled` on line 20 below?